### ISSUE TITLE
Training Loss decreases but obtaining poor performance on test set

### ISSUE THREAD

**Issue by rahul13ramesh** [Yugnaynehc/banet#11]
I am attempting to use a modified version of the code and it seems to be training successfully (Ground truth matches predicted text for training data). However the Bleu, CIDER and METEOR scores on the test set are very poor. 

I suspect it could be because of the following two factors in the modified code:

1) I are not sure if we are using the correct youtube_mapping.txt. Can you provide the same?
2) The version of the code in the repo uses C3D features, which I have attempted to remove. Can you share a version of the code which does not use C3D features?
